### ISSUE TITLE
DatePicker can't scroll on 400% zoom

### ISSUE THREAD

**Issue by jazzAlston** [microsoft/fluentui#18183]
- **Browser and OS versions**: Google Chrome /Windows 10

### Describe the issue:

We browser zoom of 400% applied the DatePicker does not apply scrollbar.

https://codepen.io/jazz8603/pen/vYxGXoE

#### Actual behavior:

User cannot see the calendar when browser zoom of 400% is applied.

#### Expected behavior:

When DatePicker is zoomed to 400% user should be able to scroll to see all content

### Documentation describing expected behavior

https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Resize_content

**Comment by gouttierre** [comment]
@jazzAlston - thank you for filing this item and providing the codepen.

@miroslavstastny / @smhigley - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by naksenzh** [comment]
Hi Team, @TristanWatanabe please advise what is the ETA for this bug?

**Comment by smhigley** [comment]
This was fixed in #18472

**Comment by jazzAlston** [comment]
Hello Sarah, this issue does not seem to be resolved.  How do I go about
reopening?
